src/github.com/getlantern/lantern-ui/app/js/services.js

```diff
@@ -205,30 +205,9 @@ angular.module('app.services', [])
       title: 'lantern-ui'
     });
 
-    function trackPageView(sessionControl) {
-      var trackers = ga.getAll();
-      for (var i =0; i < trackers.length; i++) {
-          var tracker = trackers[i];
-          if (tracker.b && tracker.b.data && tracker.b.data.w) {
-              var fields = tracker.b.data.w;
-              var gaObj = {
-                  clientId: '',
-                  clientVersion: '',
-                  language: '',
-                  screenColors: '',
-                  screenResolution: '',
-                  trackingId: '',
-                  viewPortSize: ''
-              };
-              for (var name in fields) {
-                var key = name.split(':')[1];
-                if (gaObj.hasOwnProperty(key)) {
-                    gaObj[key] = fields[name];
-                }
-              }
-              DataStream.send('Analytics', gaObj);
-          }
-      }
+    function trackPageView() {
+			console.log("SENDING PAGE VIEW")
+      ga('send', 'pageview');
```

Comment:
I just went back to this approach because it's the only thing we've seen consistently work!